tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts

```diff
@@ -154,8 +179,13 @@ suite(`Check if recommended extensions installed for ${samples}`, async function
             }
         });
 
-        test('Stopping and deleting the workspace', async function (): Promise<void> {
-            await workspaceHandlingTests.stopAndRemoveWorkspace(WorkspaceHandlingTests.getWorkspaceName());
+        test('Stop the workspace', async function (): Promise<void> {
```

Comment:
What was the reason to divide stop and delete, if it could be done at ones?

The less code in tear down steps like this, the simpler to read it and support, IMHO.

Reply:
It improves test stability. It is about the situation when uses stopAndRemoveWorkspace() method. When stopping of working workspace will be successful but deleting will be not (for example, because of unexpected page refresh, [jira issue](https://issues.redhat.com/browse/CRW-3649)) retry will starts with stopping of workspace  again what will return an error, and than all CI stage will be failed and red.

Reply:
It will be unnecessary code complexity with loops, try/catches or implicit waiters. Moreover, we already have some, but it works not every time.
https://github.com/eclipse/che/blob/666379174db85eecdef7903e47202329ef50a8c7/tests/e2e/pageobjects/dashboard/Workspaces.ts#L108

IMHO, in-build mocha retries feature is better solution here.

Reply:
I see.
It's not critical, so let it keep as it.